testing/scenario_app/android/app/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="108dp"
+    android:height="108dp"
+    android:viewportWidth="108"
+    android:viewportHeight="108">
+    <path
```

Comment:
did you draw something here? :D

Reply:
This was created by default by Android Studio.